HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -2664,6 +2715,11 @@ def construct_lognormal_income_process_unemployment(self):
         TranShkDstn : [[distribution.Distribution]]
             A list with T_cycle elements, each of which
             a discrete approximation to the transitory income shocks.
+        IncShkDstn_ntrl :  [distribution.Distribution]
+            A list with T_cycle elements, each of which is a
+            discrete approximation to the income process in a period.
+            Permanent shock component of the income process uses the permanent 
+            income neutral measure (see Harmenberg 2021).  
```

Comment:
An alternative way to design this would be to have `neutral_measure` to be an input parameter to this `construct_lognormal_income_process_unemployment` and then return the `IncShkDstn`  with the Harmenberg modification.

That would keep the number of distributions passed around more parsimonious and backwards compatible. See lines 2150-2152, which would the be unecessary.